tests/sapt-sf1/input.dat

```diff
@@ -0,0 +1,36 @@
+
+molecule dimer {
+# Set the spin of A
+0 2
+Li   0.00    0.00    0.00
+--
+# Set the spin of B
+0 2
+H   0.000   0.00   10.00
+symmetry c1
+units bohr
+}
+
+set {
+basis cc-pVDZ
+scf_type pk
+e_convergence 1e-8
+d_convergence 1e-8
+reference rohf
+}
+
+
+compare_dict = { #TEST
+    "SAPT ELST ENERGY":                 -0.02368051, #TEST
+    "E Elst10":                         -0.02368051, #TEST
+    "SAPT EXCH ENERGY":                  0.14186858, #TEST
+    "E Exch10(S^2) [highspin]":          0.14186858, #TEST
+    "E Exch10(S^2) [diagonal]":          0.00010209, #TEST
+    "E Exch10(S^2) [off-diagonal]":      0.14176648, #TEST
```

Comment:
what would you think of `SAPT EXCH10(S^2),DIAGONAL ENERGY` or similar a la https://github.com/psi4/psi4/blob/master/psi4/driver/qcdb/psivardefs.py#L93?

Reply:
Happy to do anything here, just trying out various modes.